src/lib.rs

```diff
@@ -12,6 +12,20 @@ fn jsonb_matches_schema(schema: Json, instance: JsonB) -> bool {
     jsonschema::is_valid(&schema.0, &instance.0)
 }
 
+#[pg_extern(immutable, strict)]
+fn jsonschema_is_valid(schema: Json) -> bool {
+    match jsonschema::JSONSchema::compile(&schema.0) {
+        Ok(_) => true,
+        Err(e) => {
+            notice!(
+                "Invalid JSON schema at path: {}",
+                e.instance_path.to_string()
```

Comment:
`instance_path` is not always available
```
pg_jsonschema=# select jsonschema_is_valid('"sdfg"');   
NOTICE:  Invalid JSON schema at path:               
```

could you skip the `notice` when no path is available?

Reply:
Done.